src/ray/protobuf/core_worker.proto

```diff
@@ -87,6 +87,20 @@ message DirectActorCallArgWaitCompleteRequest {
 message DirectActorCallArgWaitCompleteReply {
 }
 
+message GetObjectStatusRequest {
+  bytes owner_id = 1;
+  bytes object_id = 2;
```

Comment:
Doesn't have to happen in this PR, but we should make this
```
message Object {
   bytes owner_id = 1;
   bytes object_id = 2;
}
repeated Object objects;
```

That way, we can efficiently batch long-poll on the owner without having tons of outstanding requests.